.pre-commit-hooks.yaml

```diff
@@ -55,3 +55,9 @@
   language: python
   types: [python]
   pass_filenames: false
+- id: imagetag-branch-constraint
```

Comment:
this is a nit, and applies to all instances of `imagetag`, suggest making it 2 tokens, so `image_tag` in module names or `image-tag` in the pre-commit hook name.